index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ernest Portfolio</title>
+    <link href="styles.css" rel="stylesheet" type="text/css"/>
+</head>
+<body>
+    <!--Head Section-->
+    <header>
+        <div class="logo">
+            <h1 id="logo">My Logo</h1>
```

Comment:
- [ ] Hi. it's better to follow semantic HTML and for SEO, it's not recommended to have more than 1 `h1` tag in a file. so I'll recommend changing this h1 to `<a>` tag. Im most of the modern websites the logo is a link that redirect to the home page

Reply:
@m05e5 
Appreciate the feedback and I am working on it. The only problem I am having which has delayed the submission of my work is the Stylint. It told me, I should indent a certain portion of my CSS. I have been indenting since more and anytime I finish and submit, I get the same problem. It is frustrating me. I could use your help, please.